Earthfile

```diff
@@ -33,7 +33,8 @@ generate:
     RUN apk update && apk add openjdk11
     RUN go install go.uber.org/mock/mockgen@v0.4.0
     RUN go install github.com/princjef/gomarkdoc/cmd/gomarkdoc@latest
-    COPY (+sources/*) /src
+    COPY --dir (+lint/*) /src/
+    COPY (+tidy/*) /src/
```

Comment:
I have the same cases on membership and I start to think that generate is a middle step between tidy and lint. 

And those libs are already in the sources module No ?